usecases/config/config_handler.go

```diff
@@ -118,6 +119,25 @@ func (p Persistence) Validate() error {
 	return nil
 }
 
+// TODO: Note the mix of camelCase and snake_case
```

Comment:
remove TODO comment

Reply:
Ah yes this reminds me that I need to raise this issue. Thanks